RePoE/parser/constants.py

```diff
@@ -370,5 +370,9 @@ class ReleaseState(Enum):
     ("passive_skill_stat_descriptions.txt", "/passive_skill"),
     ("passive_skill_aura_stat_descriptions.txt", "/passive_skill_aura"),
     ("monster_stat_descriptions.txt", "/monster"),
+    ("heist_equipment_stat_descriptions.txt", "/heist_equipment"),
+    ("leaguestone_stat_descriptions.txt", "/leaguestone"),
+    ("active_skill_gem_stat_descriptions.txt", "/active_skill_gem"),
+    ("advanced_mod_stat_descriptions.txt", "/advanced_mod"),
```

Comment:
With active_skill_gem_stat_descriptions.txt now being in here, the comment above is now longer correct ("(except active_skill_gem_stat_descriptions because I don't think it is required)"). (I'm fine with it being added, was kinda weird to arbitrarily leave one file out.)